activesupport/lib/active_support/core_ext/object/blank.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "concurrent/map"
+require "ostruct"
```

Comment:
This is probably why we don't include `OpenStruct`, `ostruct` isn't required by default. In order to use `#blank?`, we now have to load `OpenStruct` only to monkey patch it.